main.go

```diff
@@ -58,6 +58,7 @@ func setupRouter() *gin.Engine {
 	sitemapRepo := sitemap.New(jobsRepo)
 	leaveRepo := leave.New(templateFS, utilsRepo)
 	notificationRepo := notification.New(templateFS, utilsRepo)
+	lifePerksImagesRepo := jobs.NewLifePerksImages(sqlDb)
```

Comment:
i think jobs already has one connection above, and as API function name is self-explanatory then no need to create one more connection

Reply:
how , we have created new repository there how it will get it

Reply:
not getting you.
if the package has the same then why do we need to create connection for every file?

Reply:
but we have used different repository here so we have to use it like this way only , that package thing i know but we have used different repository here for simplicity